templates/swagger/v1_json.tmpl

```diff
@@ -19479,6 +19489,11 @@
           "format": "int64",
           "x-go-name": "ID"
         },
+        "is_archived": {
+          "type": "boolean",
+          "x-go-name": "IsArchived",
+          "example": false
+        },
```

Comment:
I am not sure about this API change.
It will mean that if we ever want to send the archived timestamp as well, we'll need an additional field.

Reply:
![image](https://github.com/go-gitea/gitea/assets/80308335/d9884c24-14f6-4a8b-8f93-25c51258446b)

This is I have added for this.

Reply:
I know.
But AFAIK, only existing labels can be returned, this type is never used to create a label.
So we can as well return the time it was archived since only we can set it.
We can also think about adding a convenience field `is_archived` that simply uses `archived_unix.IsZero()`, but I think more important is rather the time itself.